sky/cli.py

```diff
@@ -1433,9 +1439,21 @@ def start(clusters: Tuple[str], yes: bool, idle_minutes_to_autostop: int,
       \b
       # Restart multiple clusters.
       sky start cluster1 cluster2
+      \b
+      # Restart all clusterss.
+      sky start -a
 
     """
     to_start = []
+
+    if all:
+        if len(clusters) > 0:
+            click.echo(
+                'Both --all and cluster(s) specified for sky start. '
+                'Letting --all take effect.')
```

Comment:
I have a feeling if both are specified, we should raise an error instead of printing a warning. It could be a typo and the user wanted to delete just a few clusters, but we will end up deleting all clusters

Reply:
We may want to make it consistent with other places, such as the following lines. I am ok with both, but we may want to converge on one of them. : )
https://github.com/skypilot-org/skypilot/blob/35068e65362d42fa8aacb79d4fc381ff417fb78c/sky/cli.py#L1645-L1650

Reply:
I actually ran into this myself yesterday, where I typed `sky start c2` but then realized another long-named cluster needed to be started too. So I just added `-a` at the end of the command rather than typing the long name.  I'd propose let's keep the override semantics.